### DISCUSSION TITLE
Streaming a file to rtsp-simple-server and viewing rstp

### DISCUSSION THREAD

**Discussion by JacekKac** [bluenviron/mediamtx#1208] [General]
i want to send .mp4 to rtsp server and then watch it on vlc client. 

HOSTunix
1/ rtsp-simple-server running
2/ ffmpeg -re -stream_loop -1 -i movie.mp4 -c copy -f rtsp rtsp://localhost:8554/mystream
Client Windows
VLC client rstp stream

it's working but the stream quality is veryyyyyyy poor. load of HOST is not high CPU/ram (codec "copy"), local network.

**Comment by aler9** [comment]
Hello,

> it's working but the stream quality is veryyyyyyy poor

You either have to re-encode the file (https://github.com/aler9/rtsp-simple-server#remuxing-re-encoding-compression) or use one of the available methods to reduce packet losses (https://github.com/aler9/rtsp-simple-server#corrupted-frames)

[a comment by an automated account is not shown]
